### DISCUSSION TITLE
How to disable pin reservation?

### DISCUSSION THREAD

**Discussion by RobinTail** [gpiozero/gpiozero#1175] [Q&A]
I'm making an app that operates `LED` using `gpiozero`.

And there is another app (third party), that needs to be allowed to turn that LED off when a physical button pressed.

Currently, my app is causing issues for third party app, because the `LED` always calls `pin_factory.reserve_pins()` somewhere internally, exclusively locking the LED for my app.

How to avoid that? Can I somehow disable the reservation feature?
Or is there any workaround for that?

So far, I managed to call `led.pin_factory.release_pins()` immediately after instantiation.
It kinda works. But I'd like to avoid locking at all.

**Comment by bennuttall** [comment]
The way pin control works in Bookworm is via a system of pin reservation. I'm not sure there's really a way around it. @lurch do you know?
